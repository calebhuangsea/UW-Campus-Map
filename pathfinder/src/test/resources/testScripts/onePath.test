# test two nodes with only one path

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

AddEdge graph1 n1 n2 1.1111

FindPath graph1 n1 n2