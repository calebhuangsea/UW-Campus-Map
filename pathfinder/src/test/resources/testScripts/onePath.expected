# test two nodes with only one path

created graph graph1

added node n1 to graph1
added node n2 to graph1

added edge 1.111 from n1 to n2 in graph1

path from n1 to n2:
n1 to n2 with weight 1.111
total cost: 1.111